lapis2/src/test/kotlin/org/genspectrum/lapis/config/DatabaseConfigTest.kt

```diff
@@ -27,5 +27,40 @@ class DatabaseConfigTest {
                 DatabaseMetadata(name = "pangoLineage", type = "pango_lineage"),
             ),
         )
+        assertThat(
+            underTest.schema.features,
+            containsInAnyOrder(
+                DatabaseFeature(name = "sarsCoV2VariantQuery"),
+            ),
+        )
+    }
+}
+
+@SpringBootTest(
+    properties =
+    [
+        "lapis.databaseConfig.path=src/test/resources/config/testDatabaseConfigWithoutFeatures.yaml",
+        "silo.url=http://url.to.silo",
```

Comment:
Do we need this line? This should already be present in the default config and we only want to override the other property.